### PR TITLE
User group

### PR DESCRIPTION
**Pull request by parniantaghipour**
It has been reviewed by you. I need this to be able to refer to in the other sections

### CODE REVIEW

**Comment by tjbck** [comment]
Please address all the comments, not just the ones you want to.

File: docs/features/user-group.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## Managing User Groups
+
+### Roles
+
+There are three default roles for users within a group:
+1. **Pending**: Users awaiting approval for their account to be activated.
+2. **User**: Standard users with permissions defined by the group settings.
+3. **Admin**: Users with access to all features, models, tools, and other system capabilities.
+:::note
+Admins can view the chat history of users, including those with pending access.
```
Comment (tjbck) [inline docs/features/user-group.md:18]: unless specified with the env var to disable this.

File: docs/features/user-group.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+### Groups 
+
+The **Groups** section in the Admin Panel enables administrators to:
+- **Define Access Levels**: Adjust the default permission levels for members with a **user** role.
+- **Create New User Groups**: Introduce custom user access levels with specific permissions.
+- **Assign Permissions**: Control access to workspace, tools, models, and other features for each group.
+- **Assign Users to Groups**: Assign users to specific groups.
+
+:::note
+For example, to add new tools, users must have **workspace permission** and access to **Tools**. The same applies to managing **Knowledge** and **Prompts**.
```
Comment (tjbck) [inline docs/features/user-group.md:29]: weird wording. please rewrite for clarity.

File: docs/features/user-group.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- **Define Access Levels**: Adjust the default permission levels for members with a **user** role.
+- **Create New User Groups**: Introduce custom user access levels with specific permissions.
+- **Assign Permissions**: Control access to workspace, tools, models, and other features for each group.
+- **Assign Users to Groups**: Assign users to specific groups.
+
+:::note
+For example, to add new tools, users must have **workspace permission** and access to **Tools**. The same applies to managing **Knowledge** and **Prompts**. 
+:::
+:::info
+Members with any access level (Admin, User) have access to use the tools, knowledge, and prompts.
```
Comment (tjbck) [inline docs/features/user-group.md:32]: weird wording here as well, might as well delete this part.

File: docs/features/user-group.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+   - To create a new group, click **Creat Group** and specify the group name and description.
+   - To modify an existing group, select it from the list and adjust the settings.
+
+![admin managing user groups](/img/user-group-admin.gif)
+
+3. **Assign Permissions**:
+   - Define the default permissions for users in the group by navigating to the **Groups** => **Edit User Group** => **Permissions**.
+   - Use the setting to give access to Workspace sections (Model, Tools, Promps, Knowledge), and Chat Permissions(File Upload, Chat Edit, Chat Delete, Temporary Chat).
+   :::tip 
+    Admins cannot grant users direct access to Functions, as they are advanced features that can only be managed through the admin panel.
```
Comment (tjbck) [inline docs/features/user-group.md:48]: again with the wording, this can be interpreted in an incorrect way. be more precise here.

**Review by tjbck — CHANGES_REQUESTED** [review]

File: docs/features/user-group.md (hunk 1 above)
Comment (parniantaghipour) [reply docs/features/user-group.md:18]: which env var?

Comment (tjbck) [reply docs/features/user-group.md:18]: look it up.

**Pull request closed without merging**